src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -36,11 +36,15 @@
 import java.util.Properties;
 import java.util.Set;
 import java.util.UUID;
+import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 public class HttpSinkBridgeEndpoint<K, V> extends SinkBridgeEndpoint<K, V> {
 
+    Pattern hostPattern = Pattern.compile("host=([^;]+)");
+    Pattern protoPattern = Pattern.compile("proto=([^;]+)");
```

Comment:
I think these should be case insensitive matches, if you look at the production for HTTP token in RFC 7230.